gdal/doc/source/development/rfc/rfc81_coordinate_epoch.rst

```diff
@@ -0,0 +1,92 @@
+.. _rfc-81:
+
+=============================================================
+RFC 81: Support for coordinate epochs in geospatial formats
+=============================================================
+
+============== =============================================
+Author:        Even Rouault
+Contact:       even.rouault @ spatialys.com
+Started:       2021-May-10
+Status:        Development
+Target:        GDAL 3.4
+============== =============================================
+
+Summary
+-------
+
+This RFC describes support for coordinate epochs in a few key geospatial formats
+and in GDAL/OGR API and utilities
+
+Motivation
+----------
+
+A number of coordinate reference systems (CRS) are called "dynamic CRS", that
+is the coordinates of a point on the surface of the Earth in those CRS may
+change with time. To be unambiguous the coordinates must always be qualified
+with the epoch at which they are valid. The coordinate epoch is not necessarily
+the epoch at which the observation was collected.
+
+Examples of dynamic CRS are ``WGS 84 (G1762)``, ``ITRF2014``, ``ATRF2014``.
+
+The generic EPSG:4326 WGS 84 CRS is also considered dynamic, although it is
+not recommended to use it due to being based on a datum ensemble whose positional
+accuracy is 2 metres, but prefer one of its realizations, such as WGS 84 (G1762)
+
+At time of writing, no formats handled by GDAL/OGR have a standardized way of
+encoding a coordinate epoch. We consequently have made choices how to encode it,
+admitedly not always elegant, with the aim of being as much as possible backward
+compatible with existing readers.
+Those encodings might change if corresponding official specifications
+evolve to take this concept into account. But, as this is a bit of a chicken-and-egg
+problem ("why should we care about storing coordinate epoch if no software can make use
+of it ?"), let's start with this initial solution.
+
+PROJ can handle a number of time-dependent transformations between static CRS
+and dynamic CRS, e.g a GDA2020 (static CRS/datum for Australia) to ATRF2014
+(dynamic CRS/datum for Australia), taking into account plate motion. Having
+support to store coordinate epoch will make it easier to have more accurate
+coordinate transformation.
+
+Details
+-------
+
+See https://github.com/rouault/gdal/blob/coordinate_epoch/gdal/doc/source/user/coordinate_epoch.rst
+for impacts on the API, encoding of CRS in a number of formats, and impacts on
+existing utilities.
```

Comment:
@rouault can we merge that doc with this one? Otherwise it is not completely clear what document we are reviewing and what is being approved.

Reply:
> can we merge that doc with this one? Otherwise it is not completely clear what document we are reviewing and what is being approved. If we have a single document, we can focus on one text, not two, and don't have to consider relationships between them.

I see your point but the other document is intended to be in the user documentation on gdal.org once PR 3810 is merged, and potentially be evolving over time. I don't want to go through the RFC process each time we change something for a format.